### PR TITLE
fix thermal TS error message includes area name

### PR DESCRIPTION
**Pull request by hugo-antoine-rtei**
- added area name to normalizeAndCheckNPO input variables
- used new input variable in error log message

### CODE REVIEW

**Comment by hugo-antoine-rtei** [comment]
When running the short-examples using `ctest -C Release --output-on-failure -L short-examples`,
I get fails in the short-examples-ortools-coin. The log is here: [log.txt](https://github.com/AntaresSimulatorTeam/Antares_Simulator/files/7692211/log.txt) I will try to rerun it next week to figure out what's wrong.

File: src/libs/antares/study/parts/thermal/prepro.cpp [hunk 1]
```diff
@@ -294,7 +294,7 @@ void PreproThermal::reset()
     }
 }
 
-bool PreproThermal::normalizeAndCheckNPO(const AnyString& clustername, uint clusterSize)
+bool PreproThermal::normalizeAndCheckNPO(const AnyString& clustername, uint clusterSize, const AnyString& areaname)
```
Comment (flomnes) [inline src/libs/antares/study/parts/thermal/prepro.cpp]: I recommend re-ordering the arguments of this function:

1. Area name
2. Cluster name
3. Cluster size

**Review by flomnes — CHANGES_REQUESTED** [review]

File: src/libs/antares/study/parts/thermal/prepro.cpp (hunk 1 above)
Comment (hugo-antoine-rtei) [reply src/libs/antares/study/parts/thermal/prepro.cpp]: Done !

**Comment by hugo-antoine-rtei** [comment]
Behavior was checked on a dummy study, I do get intended log outputs:
[solver-20211215-005041.log](https://github.com/AntaresSimulatorTeam/Antares_Simulator/files/7715449/solver-20211215-005041.log)
(see lines 32 & 33)

**Review by flomnes — APPROVED** [review]

**Pull request merged**